src/pages/SubmissionConfig.js

```diff
@@ -145,8 +149,8 @@ export const SubmissionConfig = ({ methodId }) => {
         onChangeTab: v => setActiveTab({ key: v }),
         finalStep: h(ButtonPrimary, {
           style: { marginLeft: '1rem' },
-          // disabled: !!Utils.computeWorkspaceError(ws) || !!noLaunchReason || currentSnapRedacted || !!snapshotReferenceError,
-          // tooltip: Utils.computeWorkspaceError(ws) || noLaunchReason || (currentSnapRedacted && 'Workflow version was redacted.'),
+          disabled: _.isEmpty(selectedRecords),
```

Comment:
I kept trying to submit my workflow without any records selected, which errored out. I figured it would be better to just have this button disabled if no records are selected.